specification/purview/data-plane/Azure.Analytics.Purview.Scanning/preview/2022-02-01-preview/scanningService.json

```diff
@@ -0,0 +1,7565 @@
+    },
+    "AzureMySqlSystemScanRuleset": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/SystemScanRuleset"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "allOf": [
+            {
+              "$ref": "#/definitions/AzureMySqlScanRulesetProperties"
+            }
+          ]
+        }
+      },
+      "x-ms-discriminator-value": "None"
+    },
+    "AzureStorageSystemScanRuleset": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/SystemScanRuleset"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "allOf": [
+            {
+              "$ref": "#/definitions/AzureStorageScanRulesetProperties"
+            }
+          ]
+        }
+      },
+      "x-ms-discriminator-value": "AzureStorage"
+    },
+    "TeradataSystemScanRuleset": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/SystemScanRuleset"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "allOf": [
+            {
+              "$ref": "#/definitions/TeradataScanRulesetProperties"
+            }
+          ]
+        }
+      },
+      "x-ms-discriminator-value": "None"
+    },
+    "OracleSystemScanRuleset": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/SystemScanRuleset"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "allOf": [
+            {
+              "$ref": "#/definitions/OracleScanRulesetProperties"
+            }
+          ]
+        }
+      },
+      "x-ms-discriminator-value": "None"
+    },
+    "SapS4HanaSystemScanRuleset": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/SystemScanRuleset"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "allOf": [
+            {
+              "$ref": "#/definitions/SapS4HanaScanRulesetProperties"
+            }
+          ]
+        }
+      },
+      "x-ms-discriminator-value": "None"
+    },
+    "SapEccSystemScanRuleset": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/SystemScanRuleset"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "allOf": [
+            {
+              "$ref": "#/definitions/SapEccScanRulesetProperties"
+            }
+          ]
+        }
+      },
+      "x-ms-discriminator-value": "None"
+    },
+    "PowerBISystemScanRuleset": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/SystemScanRuleset"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "allOf": [
+            {
+              "$ref": "#/definitions/PowerBIScanRulesetProperties"
+            }
+          ]
+        }
+      },
+      "x-ms-discriminator-value": "None"
+    }
+  },
+  "parameters": {
+    "api-version": {
```

Comment:
This should have x-ms-parameter-location: client

Reply:
added `x-ms-parameter-location: client`